Chiazokam/Politico
# #163504424 create the get one office endpoint

#### What does this PR do?
This PR tests and creates the get one office Endpont

#### Description of Task to be completed?
- write tests for every test case
- implement admin check with middleware
- create the get one office endpoint

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-database-admin-get-one-office-163504424`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Send a `GET` request to `http://localhost:3000/api/v1/offices/1`
- You should receive a successful response

#### Any background context you want to provide?
Nil

#### What are the relevant pivotal tracker stories?
[#163504424](https://www.pivotaltracker.com/story/show/163504424)

### Images
![image](https://user-images.githubusercontent.com/26940294/51908112-2bb21300-23c9-11e9-8a8f-4e29bf96ebc3.png)
